hugegraph-api/src/main/java/com/baidu/hugegraph/server/ApplicationConfig.java

```diff
@@ -39,10 +43,21 @@
 import com.baidu.hugegraph.define.WorkLoad;
 import com.baidu.hugegraph.event.EventHub;
 import com.baidu.hugegraph.util.E;
+import com.baidu.hugegraph.version.CoreVersion;
 import com.codahale.metrics.MetricRegistry;
 import com.codahale.metrics.jersey3.InstrumentedResourceMethodApplicationListener;
 
 @ApplicationPath("/")
+@OpenAPIDefinition(
+        info =
+        @Info(
+                title = "HugeGraph RESTful API",
+                version = CoreVersion.DEFAULT_VERSION,
+                description = "All management API for HugeGraph",
+                contact =
+                @Contact(
+                        url = "https://github.com/apache/incubator-hugegraph",
```

Comment:
We can use this address `https://github.com/apache/hugegraph` and  it will be automatically redirected.

Reply:
Done.
<img width="1232" alt="image" src="https://user-images.githubusercontent.com/16890042/168518436-c6856fd8-7a0d-40ea-81e8-ab781569456a.png">